### DISCUSSION TITLE
development setup Issue

### DISCUSSION THREAD

**Discussion by TakshPanchal** [trypromptly/LLMStack#142] [Q&A]
I am not able to find `docker-compose` file described in this [Development guideline](https://llmstack.ai/docs/development/introduction). Any suggestions for setting up dev environment?

**Comment by ajhai** [comment]
Sorry about that. Can you follow the setup instructions at https://llmstack.ai/docs/getting-started/development?

**Comment by TakshPanchal** [comment]
I am having issues with `poetry install` command.

**Reply by TakshPanchal** [reply]
<img width="1291" alt="image" src="https://github.com/trypromptly/LLMStack/assets/52617262/57e85acd-9157-4360-87c1-cf5b546f4223">

**Comment by TakshPanchal** [comment]
Also @ajhai , After running pre-commits I am getting lots of changes in the repo, which I yesterday recently branched from the `main` branch.

**Reply by ajhai** [reply]
@TakshPanchal  Please rebase your branch. It should all be fixed now. Let me know if you are still seeing issues.

**Reply by TakshPanchal** [reply]
I am still getting the same error.
